css/main.css

```diff
@@ -1,3 +1,89 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+:root {
+    --beige: #E6DACD;
+    --lbeige: #F4ECE6;
+    --blue: #0150FD;
+    --white: #FFF;
+    --black: #000;
+}
+
+body{
+    font-family: 'Quicksand', sans-serif;
+    background-color: var(--white);
+}
+
+h3,b{
+    font-family: 'Poppins', sans-serif;
+    font-size: 27px;
+}
+h1{
+    font-family: 'Poppins', sans-serif;
+    font-size: 100px;
+}
+
+.navbar{
+    background-color:var(--white);}
+
+#navholder{
+    height: 80px;
+}
+.hero{
+    height:800px ;
+    z-index: 2;
+    position: relative;
+    
+}
+#background {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 42%;
+    height: 900px;
+    background-color: var(--beige);
+    z-index: 1;
+}
```

Comment:
Don't use IDs in your CSS. You can use IDs for JavaScript, but when adding styles to HTML, target your elements using classes.